Consul.AspNetCore/Consul.AspNetCore.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <VersionPrefix>0.0.1</VersionPrefix>
+    <VersionPrefix>3.1.8</VersionPrefix>
```

Comment:
I am not sure should it be, but some projects using same version (MassTransit, Quartz, OpenTelemetry, Microsoft.*) for DI extensions to mark them "as compatible"

Also using same version is the case when extensions has same release cycle

Discussable

Reply:
OK, so we have two options:
- Same release cycle -> same version for `Consul` and `Consul.ApnNetCore`
- Independent release cycle -> we can use independent version for `Consul` and `Consul.ApnNetCore`

I'm leaning towards independent release cycle as I don't expect `Consul.ApnNetCore` to have a lot of changes (implying a lot of new versions published). And changes to `Consul` library are unlikely to cause any changes to the `Consul.ApnNetCore` so with independent release cycles we will have much fewer versions of `Consul.ApnNetCore`.

Same release cycle is also fine. Maybe we will end up with more versions of `Consul.ApnNetCore` , but that is not a big deal after all. However, it would be good idea to move `VersionPrefix` into `Directory.Build.props` file so same `VersionPrefix` is use consistently for all projects in this repo.

Reply:
I bet on independant release cycle, then =)

1. Consul.AspNetCore will not change often
2. I plan to add separate workflow with independant pipeline for CI to this integrations